tests/parsing/LineParserTest.py

```diff
@@ -13,13 +14,24 @@ def test_empty_line(self):
         self.check_data_set('\n \n \n')
 
     def test_comment_parsing(self):
+        logger = logging.getLogger()
         self.check_data_set('# comment only$§\n',
                             output_comment='# comment only$§')
         self.check_data_set('   ; comment only  \n',
                             output_comment='; comment only')
         self.check_data_set('   ; \\comment only  \n',
                             output_comment='; comment only')
         self.check_data_set('#', output_comment='#')
+        with self.assertLogs(logger, 'WARNING') as cm:
+            self.check_data_set('##\n',
+                                output_comment='##')
+        self.assertEquals(cm.output, "The comment line does not have \
```

Comment:
See that this statement should be indented inside the `with` block.
As soon as the with block ends, the `logger as cm` also stops.
I think that could be the problem here.. :)